### ISSUE TITLE
Agent's version number should be kept in one file instead of in each production .csproj file

### ISSUE THREAD

**Issue by SergeyKleyman** [elastic/apm-agent-dotnet#159]
Currently Agent's version number is repeated in in each production .csproj file a few times:
```
<AssemblyVersion>0.0.2.0</AssemblyVersion>
<InformationalVersion>0.0.2.0</InformationalVersion>
<FileVersion>0.0.2.0</FileVersion>
<PackageVersion>0.0.2.0-alpha</PackageVersion>
```

Instead we would like to have just in one file containing all the Agent's version number related information without any repetitions.

**Comment by gregkalapos** [comment]
A minor thing: those 4 values mean 4 different things. Probably we'll be good to go with the same for all of those - but before we fix all of them we should make sure if that's ok.

**Comment by vhatsura** [comment]
@gregkalapos, I believe it can be closed due to it was implemented in #588
